### PR TITLE
Lines comment feature in diff view

### PR DESCRIPTION
**Pull request by makhov**
A simple «Code Review» feature. Comment lines in diff view:
line-comment

### CODE REVIEW

**Comment by makhov** [comment]
It's in beta now. You can only create comments. I'll add editing and removing soon.

**Comment by Unknwon** [comment]
Wow, a lot changes, thanks!

Would you like I create a `comment` branch to merge this since changes are so many.

**Comment by makhov** [comment]
It's up to you. Just in case I already merge actual dev branch my one.

**Comment by Unknwon** [comment]
New `comment` branch is created.

**Comment by makhov** [comment]
Note, I commit some updates.

File: cmd/web.go [hunk 1]
```diff
@@ -444,7 +444,10 @@ func runWeb(ctx *cli.Context) {
 			m.Get("/src/*", repo.Home)
 			m.Get("/raw/*", repo.SingleDownload)
 			m.Get("/commits/*", repo.RefCommits)
```
Comment (phsmit) [inline cmd/web.go]: Comment on old line 446

File: cmd/web.go [hunk 2]
```diff
@@ -444,7 +444,10 @@ func runWeb(ctx *cli.Context) {
...
+			m.Get("/commit/comment/*", repo.GetCommentForm)
```
Comment (phsmit) [inline cmd/web.go]: Comment on new line 447

**Comment by Unknwon** [comment]
Hi there, I've created `comment` branch, please re-PR to it: https://github.com/gogits/gogs/tree/comment

Thanks!

**Pull request closed without merging**